makefile

```diff
@@ -25,7 +25,10 @@ setup:
 	@venv/bin/pip3 install -r requirements.txt
 
 preview:
-	flask --app ops_eng_app/__init__ --debug run
+	flask --app application/__init__ --debug run
+
+app:
```

Comment:
Can you rename this to make it more descriptive? Perhaps `local-app` or `local` Something in that ball park.